src/middleware/adminAuth.ts

```diff
@@ -0,0 +1,13 @@
+const ADMIN = {
+  email: 'test@example.com',
+  password: 'password',
+};
+
+const adminAuth = (email, password) => {
```

Comment:
Just a minor issue, guard clauses should catch errors and the final return statement should be the success scenario: https://nus-cs2103-ay2021s1.github.io/website/se-book-adapted/chapters/codeQuality.html#make-the-happy-path-prominent

Reply:
Fixed. Will merge if this is correct way to use guard clauses